File: INICIANTE/1116/Main.java
import java.util.Scanner;

public class Main {

    static void formattedPrint(int x, int y) {
        if (y != 0)
            System.out.printf("%.1f\n", (double) x / y);
        else
            System.out.println("divisao impossivel");
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        while (n-- > 0) {
            int x = scanner.nextInt();
            int y = scanner.nextInt();
            formattedPrint(x, y);
        }
        scanner.close();
    }
}

File: INICIANTE/1116/main.c
#include <stdio.h>

void formattedPrint(int x, int y) {
    if (y)
        printf("%.1lf\n", (double)x / y);
    else
        printf("divisao impossivel\n");
}

int main() {
    int n, x, y;
    scanf("%d", &n);
    while (n--) {
        scanf("%d %d", &x, &y);
        formattedPrint(x, y);
    }
    return 0;
}
